examples/scenes/src/test_scenes.rs

```diff
@@ -53,6 +61,7 @@ macro_rules! scene {
 }
 
 export_scenes!(
+    #[cfg(feature = "cosmic_text")] cosmic_text_scene(cosmic_text),
     splash_with_tiger(impls::splash_with_tiger(), "splash_with_tiger", false),
```

Comment:
The splash screen should still be the first screen, to show the controls to anyone.

Reply:
Done